test/fft/test_spectral_op.py

```diff
@@ -167,10 +167,8 @@ def test_check_output(self):
 
     def test_check_grad(self):
         self.check_grad(
-            "X",
+            ["X"],
             "Out",
-            user_defined_grads=[self.x_grad],
-            user_defined_grad_outputs=[self.out_grad],
```

Comment:
shall we delete `self.x_grad` and `self.out_grad` above together?

Reply:
It can be deleted now, and it will support random out_grad later and add  self.out_grad back